src/ui/windows/TogglDesktop/TogglDesktop/ui/controls/TogglNotification.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+
+namespace TogglDesktop
+{
+    public class TogglNotification : Control

```

Comment:
```suggestion
    public class TogglNotification : ContentControl 
```
I think its more convenient to derive from ContentControl instead of Control and then you don't need to implement Content  dependency property

Reply:
Agree, thx 👍